commands/api.ts

```diff
@@ -26,10 +26,15 @@ export const kibanaFetch = async (url, params, apiVersion = 1) => {
       headers: headers,
       ...params,
     });
-    const data = await result.json();
-    if(data.statusCode && data.statusCode !== 200) {
+    const data: unknown = await result.json();
+    if (!data || typeof data !== 'object') {
+	    throw new Error;
+    }
+
+    if('statusCode' in data && data.statusCode !== 200) {
       console.log(data)
-      throw new Error(data.message)
+      // TODO
+      throw new Error((data as any).message)
```

Comment:
We have a lot of issues with untyped stuff. I think some tool to validate and type stuff would be good.

Reply:
Yea I can't what is cool these days. io-ts I think? I found that library hard to understand from the docs, but easy-ish to understand when looking at code in Kibana. 

For the Event Analyzer aka Resolver, which is the last time I worked on a feature, I wrote my own: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/resolver/models/schema.test.ts

Reply:
I used io-ts, it meant I could delete a load of code which is nice

https://github.com/elastic/security-documents-generator/pull/9/commits/f08439e42bd43ec69e6536504f31a257f2aea175

Reply:
For the record, my comment was more tongue-in-cheek. I understood Robert's comment as jaust generally typing stuff, hence the switch to Typescript.
I did not realise this was about schemas and boundary layer types. 

I'm really happy we're using io-ts for this! Superb job @hop-dev